linker/Linker/Driver.cs

```diff
@@ -72,7 +74,71 @@ public static int Execute (string[] args, ILogger customLogger = null)
 
 		public Driver (string [] args)
 		{
-			_queue = new Queue<string> (args);
+			_queue = ProcessResponseFile (args);
+		}
+
+		Queue<String> ProcessResponseFile (string [] args)
+		{
+			var result = new Queue<string> ();
+			foreach (string arg in args) {
+				if (arg.StartsWith ("@")) {
+					try
+					{
+						string responseFileName = arg.Substring(1);
+						IEnumerable<string> responseFileLines = File.ReadAllLines(responseFileName).Select(line => line.Trim());
```

Comment:
Please do it without reading the whole file first. Simple `while ((line = responseFileName.ReadLine ()) != null) {` would work better

Reply:
Will be fixed in the new iteration.